plugins/modules/dcnm_network.py

```diff
@@ -1322,7 +1345,14 @@ def get_diff_merge(self, replace=False):
                         if not net_id_obj['DATA']:
                             continue
 
-                        net_id = net_id_obj['DATA'].get('segmentId')
+                        if self.dcnm_version == 11:
```

Comment:
Why is the version 11 key now `partitionSegmentId` instead of just `segmentId`?

Reply:
This should be segmentId. This got changed during final formatting before commit. Changed back to segmentId